src/Components/HomePage.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { FC, useState } from 'react'
+import {
+    Box,
+    List,
+    Snackbar
+} from '@material-ui/core';
+
```

Comment:
I suggest removing this line.

Reply:
Resolved on 4f217aae31899960ed45d358ca29d0de1f1d8135